scripts/assets.sh

```diff
@@ -0,0 +1,21 @@
+# !/usr/bin/bash
+
+set -e -x
+
+NODE_ENV=production
+rm -rf public
+mkdir public
+mkdir public/assets
+browserify \
+  $(find desktop/assets mobile/assets -name '*.coffee') \
+  -t babelify \
+  -t caching-coffeeify \
+  -t jadeify \
+  -t uglifyify \
+  -p [ factor-bundle -o 'uglifyjs | gzip > public/assets/`basename $FILE .coffee`.js.jgz' ] \
+  | uglifyjs | gzip > public/assets/common.js.jgz
+stylus \
+  $(find desktop/assets mobile/assets -name '*.styl') \
+  --compress \
+  -o public/assets
+gzip -S .cgz $(find public/assets -name '*.css')
```

Comment:
It seems nicer to be able to control the asset pipeline directly from within Force